source/coding-the-legislation/35_periods.md

```diff
@@ -21,16 +16,17 @@ In OpenFisca inputs, periods are encoded in strings. All the valid period format
 |-------------------|-----------------|---------------------|--------------------------------------------------|-----------------------------------------------------------------------|
 | `AAAA`            | Calendar year   | `'2010'`            | The year 2010.                                   | From the 1st of January 2010 to the 31st of December 2010, inclusive. |
 | `AAAA-MM`         | Month           | `'2010-04'`         | The month of April 2010.                         | From the 1st of April 2010 to the 30th of April 2010, inclusive.      |
+| `AAAA-MM-DD`      | Day             | `'2010-04-06'`      | The day of the 6th of April 2010.                | The whole day of the 6th of April 2010.      |
```

Comment:
Not sure if "The day of the 6th of April 2010. " works. 

6th April 2010 (GB) ? April 6th, 2010 (US) ?